src/impact-dashboard/EpidemicModelContext.tsx

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+
+const {
+  populationAndHospitalData,
+} = require("../page-overview/assets/dataSource") as any;
```

Comment:
- Can you move the `dataSource` file up into a more common folder? I'm thinking into `infection-model`. Or can make another top-level folder called `data` or something like that.
- You can rename the file to `.tsx`. You won't have to change anything about the contents of the file, but also you'll now be able to `import` it and not use `any`. 

We don't have to do this in this PR, but I feel like it wouldn't be too hard for a change to make.

Reply:
I agree but we need to do a whole new pass at the external datasources to pull in more stuff for this model so I left it alone for now. would rather keep this PR small